web/wp-content/plugins/ad-stack/inc/ads-global-targeting.php

```diff
@@ -32,17 +32,31 @@ function ads_global_targeting_parameters() {
 	$url_part = $unslash_url;
 
 	$g_targeting = array(
-		'property' => '6178',
-		'siteId'   => 'cpt',
-		'pageType' => $page_type,
-		'urlPath'  => $url_part,
-		'keyWords' => get_post_tags(),
-		'category' => $top_level_categories,
-		'topic'    => $sub_categories,
+		'property'   => '6178',
+		'siteId'     => 'cpt',
+		'pageType'   => $page_type,
+		'urlPath'    => $url_part,
+		'keyWords'   => get_post_tags(),
+		'category'   => $top_level_categories,
+		'topic'      => $sub_categories,
+		'breakpoint' => get_breakpoints(),
 	);
 	printf( PHP_EOL . '<script type="text/javascript"> var tmbi_ad_data = %s </script>' . PHP_EOL, wp_json_encode( $g_targeting ) );
```

Comment:
This should use [wp_localize_script](https://developer.wordpress.org/reference/functions/wp_localize_script/). What's the reason not to?